pkg/transport/generic_transport_controller.go

```diff
@@ -452,6 +509,23 @@ func (c *genericTransportController) syncProperties(ctx context.Context, invName
 	}
 }
 
+func (c *genericTransportController) syncCustomTransform(ctx context.Context, name string) error {
+	logger := klog.FromContext(ctx)
+	ct, err := c.customTransformLister.Get(name)
+	if errors.IsNotFound(err) {
+		ct = nil
+	} else if err != nil { // This never happens; retry will not help
+		logger.Error(err, "Failed to Get from CustomTransformLister", "name", name)
+		ct = nil
+	}
+	c.customTransformCollection.NoteCustomTransform(ctx, name, ct)
+	return nil
+}
+
+func ctSpecGroupResource(spec v1alpha1.CustomTransformSpec) metav1.GroupResource {
```

Comment:
This function is declared here, but used only within `custom_transform_collection.go`. Perhaps it should be moved to there?

Reply:
BTW, in pkg code, only test files use snake case names, while other code file names use lisp case. Both `generic_transport_controller.go` and `custom_transform_collection.go` should use lisp case, or the rest of the files should be changed to use snake case.

Reply:
I do not think that `pkg` is or should be special in this regard. I have seen enough of both dash and underscore that I did not feel the need to try to enforce some consistency. It would feel a bit awkward to me to enforce consistent usage of the opposite of what the code generators create.

Reply:
I think we should follow one pattern. If you think we should follow what the generators create, then we will. But not being uniform in this matter looks bad I believe (and personally bothers me). You don't have to do this of course, I can make a followup PR.